ferretdb/ferretdb.go

```diff
@@ -102,11 +104,21 @@ func New(config *Config) (*FerretDB, error) {
 	}
 
 	lOpts := &logging.NewHandlerOpts{
-		Base:       "console",
-		Level:      logLevel,
 		SkipChecks: true,
+		Level:      logLevel,
+	}
+
+	var logger *slog.Logger
+
+	if config.Logger == nil {
+		lOpts.Base = "console"
+		logger = logging.WithName(logging.Logger(logOutput, lOpts, ""), "ferretdb")
+	} else {
+		handler := config.Logger.Handler()
```

Comment:
If I set config.Logger with a logger that configured with log level such as `logger.Enabled(ctx, slog.LevelDebug)`, how is it work?

I think it should set `NewHandlerOpts.Level` with the configured level of the `config.Logger`. Could it be tested?

Reply:
I see your point, the simplest way is using `LogLevel slog.Leveler` configuration option. Otherwise, you would need to figure out the level by calling `Enabled()` to determine the level set by the logger.

Reply:
@chilagrow I think we can use `LogLevel slog.Level` configuration option, how about create a new handler named `embeddableHandler` in `logging` package, and make the `custom logger.Handler` as a field in `embeddableHandler`? Then we can use LogLevel for the `custom logger`